src/app/Shared/Services/Report.service.tsx

```diff
@@ -130,3 +219,18 @@ export const isGenerationError = (toCheck: any): toCheck is GenerationError => {
   }
   return true;
 };
+
+export interface RuleEvaluation {
+  name: string;
+  description: string;
+  score: number;
+  topic: string;
+}
+
+export const RED_SCORE_THRESHOLD = 75;
```

Comment:
Might be cleaner to use enum here for these scores?

Reply:
Not sure, I still need them to compare numbers directly (e.g. `if score < RED_SCORE_THRESHOLD, then` etc...)

Reply:
I think we can try Numeric Enum: https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums and use comparison as normal: https://bobbyhadz.com/blog/typescript-compare-enums

What do you think?